makefile

```diff
@@ -7,6 +7,7 @@ help:
 	@echo "  test        runs tests"
 	@echo "  compile     sets up your js files for production"
 	@echo "  serve       run the webserver"
+	@echo "  shrinkwarp  resets the shrinkwrap.json file"
```

Comment:
@Alduhoo Small mistake, sorry: It says `shrinkwarp`, not `shrinkwrap` :D can you just fix this typo and I'll auto-merge?